docs/docker-compose/README.md

```diff
@@ -111,7 +116,7 @@ suited for test or demo environment but certainly not for production use.
 ## Prerequisites
 
 As well as the prerequisites mentioned on the [main
-README](/README.md#prerequisites) you will need a machine with at least 13GB of
+README](/index.html#prerequisites) you will need a machine with at least 13GB of
```

Comment:
use relative path to .md, the plugin should do the magic

Reply:
This approach doesn't work, don't know why: https://github.com/Alfresco/acs-deployment/actions/runs/9061712638/job/24894070823#step:7:15